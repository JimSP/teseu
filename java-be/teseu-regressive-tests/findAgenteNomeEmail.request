GET /agente/findAgenteNomeEmail
accept:application/json
Content-Type:application/json
token:$token
filtro:$filtro