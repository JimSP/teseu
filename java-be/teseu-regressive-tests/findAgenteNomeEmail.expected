200=$anuncio.httpStatus
9728=$agentes.responseBody.idCliente
SÃ£o Paulo=$agentes.responseBody.nomeCidade
AGENTE JIRA=$agentes.responseBody.nomeCliente
0=$agentes.responseBody.qtdAnunciosAtivos
5=$agentes.responseBody.qtdAnunciosPublicoAgente
0=$agentes.responseBody.cliente.qtdAnunciosPublicoAgenteAtivo
4=$agentes.responseBody.cliente.qtdPaginas
0=$agentes.responseBody.cliente.qtdPedido
SP=$agentes.responseBody.cliente.ufEstado